Gordon360/Services/MembershipService.cs

```diff
@@ -356,41 +293,41 @@ public void TogglePrivacy(int membershipID, bool isPrivate)
         /// </summary>
         /// <param name="membership">The membership to validate</param>
         /// <returns>True if the membership is valid. Throws ResourceNotFoundException if not. Exception is caught in an Exception Filter</returns>
-        private async Task<bool> ValidateMembershipAsync(MEMBERSHIP membership)
+        public async Task<bool> ValidateMembershipAsync(MembershipUploadViewModel membership)
         {
-            var personExists = _context.ACCOUNT.Where(x => x.gordon_id.Trim() == membership.ID_NUM.ToString()).Any();
+            var personExists = _context.ACCOUNT.Where(x => x.AD_Username == membership.Username).Any();
             if (!personExists)
             {
                 throw new ResourceNotFoundException() { ExceptionMessage = "The Person was not found." };
             }
```

Comment:
Would it make sense to use the AccountService here?

Reply:
Good call.